Sources/StreamChat/ChatClient_Tests.swift

```diff
@@ -858,6 +862,7 @@ class ChatClient_Tests: StressTestCase {
             eventWorkerBuilders: [],
             environment: testEnv.environment
         )
+        client.connectAnonymousUser()
```

Comment:
why this change?

Reply:
We need to connect before we can simulate going to background. Basically all the existing tests that test the way we handle config settings and implementation details of going background/active now need to connect first